Src/CSharpier.Tests/TestFiles/FieldDeclaration/FieldDeclarations.cst

```diff
@@ -51,6 +51,10 @@ public class ClassName
         SomeLongTypeName,
         SomeOtherLongTypeName_______________________________
     >();
+
+    private static readonly Func<SyntaxTrivia, bool> s_isVisualBasicCommentTrivia = (
```

Comment:
Do people really use Hungarian notation (e.g. `s_`) in C#? :open_mouth:

Reply:
Not really. I just picked up this example from the linked issue.

Reply:
I've definitely run into it, which is why it ended up in the linked issue.